### PR TITLE
Optimize layers

### PR DESCRIPTION
**Pull request by Fokko**
Hi,

To reduce the footprint of the image, I've merged some of the layers. Because of the way how UnionFS works, all the files that are created in a layer, and deleted in a layer below, will still be shipped with the container, but does not carry any value. When merging some of the layers, the creation and deletion of the files is within a single layer, and therefore it will not be included in the layer.

Cheers, Fokko

### CODE REVIEW

**Comment by Fokko** [comment]
Resolved the conflicts 👍

**Comment by gianm** [comment]
👍

File: Dockerfile [hunk 1]
```diff
@@ -1,26 +1,24 @@
 FROM ubuntu:14.04
 
+# Set version and github repo which you want to build from
+ENV GITHUB_OWNER druid-io
+ENV DRUID_VERSION 0.9.2
```
Comment (gianm) [inline Dockerfile]: If you meant this to be a Druid release version, it should be `druid-0.9.2`. "0.9.2" is the corresponding release branch but not necessarily the same thing.

Comment (gianm) [inline Dockerfile]: Although `druid-0.10.0` would be more better and newer :)

**Comment by gianm** [comment]
Lol I'm really good at reviewing

**Pull request merged**